packages/sdk/src/utils/calldata.ts

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable no-bitwise */
+
+import type { Hex } from 'viem'
+
+/**
+ *  typescript adaptations of cdCompress and cdDecompress from
+ *  https://github.com/Vectorized/solady/blob/main/js/solady.js
+ *
+ * these are custom calldata compression and decompression functions
+ * that are supported by Sound contracts (sound creator v2, sound edition v2, and super minter)
+ * and can reduce calldata size (biggest influence on L2 gas costs) when there are lots of zeros or lots of 0xffs
+ */
+export function cdCompress(original: Hex): Hex {
```

Comment:
@PabloSzx , is this exported such that I can delete the code in code repo and import from here to dedup?

Reply:
Yeah it would be available with a deep import, but if you also re export from index it would be cleaner but no preference